<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomLayerButton</class>
 <widget class="QWidget" name="CustomLayerButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widBody" native="true">
     <property name="styleSheet">
      <string notr="true">#widBody{
 	background-color: rgb(182, 182, 182);
}

#widBody:hover{
 	background-color: rgb(200, 200, 200);
}

#widBody[selected = true]{
	background-color: rgb(220, 220, 220);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblLayerName">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #101010;
background:none;</string>
        </property>
        <property name="text">
         <string>LAYER 01</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnColor">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnShow">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
	border-image: url(:/resource/hide.png);
	background:none;
}

QToolButton:hover{
	border-image: url(:/resource/hide_hover.png);
}

QToolButton:pressed{
	border-image: url(:/resource/hide.png);
}
                       
QToolButton:checked{
	border-image: url(:/resource/visibility.png);
}

QToolButton:checked:hover{
	border-image: url(:/resource/visibility_hover.png);
}

QToolButton:checked:pressed{
	border-image: url(:/resource/visibility.png);
}                                                     </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnUnlock">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
	border-image: url(:/resource/locked.png);
	background:none;
}

QToolButton:hover{
	border-image: url(:/resource/locked_hover.png);
}

QToolButton:pressed{
	border-image: url(:/resource/locked.png);
}
        
QToolButton:checked{
	border-image: url(:/resource/unlocked.png);
}

QToolButton:checked:hover{
	border-image: url(:/resource/unlocked_hover.png);
}

QToolButton:checked:pressed{
	border-image: url(:/resource/unlocked.png);
}
                                                                                                                                                </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
